README.md

```diff
@@ -475,6 +477,19 @@ log(isOver6Feet(7).isErr()) // false
 log(isOver6Feet(4).isErr()) // true
 ```
 
+##### Result.value
+Returns either the value contained in the Ok, or the error in the Err
+
+```js
+import { Ok, Err } from 'pratica'
+
+const six = Ok(6).value()
+const error = Err(new Error("Test Error")).value()
```

Comment:
Let's not use new Error in the examples, could get confusing

Reply:
Fixed in commit e8f5400f54471f7d1a9d2ebbdbe553aad6fced65